### ISSUE TITLE
VoiceOver not properly tracking focus for Mac Desktop IDE

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#4862]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.3.611
    OS Version      : macOS 10.15.1
    R Version       : 3.5.1

### Steps to reproduce the problem
- Start VoiceOver
- Start RStudio desktop
- ensure `Help / Accessibility / Screen Reader Support` is checked

### Describe the problem in detail
- Many (most?) times when the IDE starts, VoiceOver doesn't know that focus has been placed in the RStudio console, and thus it doesn't announce this fact, it usually just says "RStudio, Group"
- Typing into the console in this state also announces nothing
- Clicking mouse on console or other actions with keyboard that change focus often will nudge VoiceOver into knowing what's going on, but that's not acceptable behavior
- Lots of other weirdness, for example, Tab controls are announced as Radio Buttons

### Describe the behavior you expected
- Should announce that focus is in Console (as it does with RStudio Server); for example:
```
Console, Edit Text
Insertion at beginning of text
```

**Comment by gtritchie** [comment]
Opened a Qt issue, as I can repro this with their QtWebEngine samples:

https://bugreports.qt.io/browse/QTBUG-78284

**Comment by gtritchie** [comment]
The situation was somewhat improved by this: #5751 

Additional improvements would require a custom Qt build (or a future Qt release that incorporates changes as seen in https://codereview.qt-project.org/c/qt/qtwebengine/+/281210

**Comment by gtritchie** [comment]
Accessibility in 1.4 is for server only so removing this from 1.4 milestone.

**Comment by gtritchie** [comment]
Early prototype testing confirms that this will be fixed with the move to Electron. Keeping open until we have production code available for testing.

**Comment by gtritchie** [comment]
Confirmed this is fixed by Electron. I tested with a local build using Electron 15.3, on macOS 12.0.1, and VoiceOver is tracking focus as expected.

**Comment by gtritchie** [comment]
VoiceOver now announcing as expected in RStudio Desktop (Electron). Tried build 2022.06.0-daily+244 on macOS 12.3.
